src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,75 @@
 
-## 0134-0143 - Title
+## 0134-0143 – Title
 
-Title of the game in UPPER CASE ASCII. If it is less than 16 characters
-then the remaining bytes are filled with $00 bytes. When inventing the CGB,
-Nintendo has reduced the length of this area to 15 characters, and some
-months later they had the fantastic idea to reduce it to 11 characters
-only. The new meaning of the ex-title bytes is described below.
+These bytes contain the title of the game in upper case ASCII.
+If the title is less than 16 characters long, the remaining bytes all have the
+value `$00`.
 
-## 013F-0142 - Manufacturer Code
+While developing the CGB, Nintendo reduced the length of this area
+to 15 characters, and some months later they had the fantastic idea to reduce it
+further to just 11 characters. The new meanings of the bytes `$013F`-`$0143` are
+described below.
 
-In older cartridges this area has been part of the Title (see above), in
-newer cartridges this area contains an 4 character uppercase
-manufacturer code. Purpose and Deeper Meaning unknown.
+## 013F-0142 – Manufacturer Code
 
-## 0143 - CGB Flag
+In older cartridges these bytes were part of the Title (see above).
+In newer cartridges they contain a 4-character manufacturer code
+(uppercase). The purpose is unknown.
 
-In older cartridges this byte has been part of the Title (see above). In
-CGB cartridges the upper bit is used to enable CGB functions. This is
-required, otherwise the CGB switches itself into Non-CGB-Mode. Typical
-values are:
-```
- 80h - Game supports CGB functions, but also works on old Game Boys.
- C0h - Game works on CGB only (physically the same as $80).
-```
-Values with Bit 7 set, and either Bit 2 or 3 set, will switch the
+## 0143 – CGB Flag
+
+In older cartridges this byte was part of the Title (see above). In
+CGB cartridges the most significant bit is used to enable CGB functions. This is
+required; otherwise the CGB switches itself into Non-CGB-Mode.
+
+Typical values are:
+
+Code  | CGB support
+------|------------------------------------------------------------------
+`$80` | The game supports CGB functions (but also works on old Game Boys)
+`$C0` | The game works on CGB only (physically the same as `$80`)
+
+Values with bit 7 set, and either bit 2 or 3 set, will switch the
 Game Boy into a special non-CGB-mode called "PGB mode".
 
 TODO: research and document PGB modes...
 
-## 0144-0145 - New Licensee Code
+## 0144-0145 – New Licensee Code
 
-Specifies a two-character ASCII licensee code, indicating the company or
-publisher of the game. These two bytes are used in newer games only
-(games that have been released after the SGB has been invented). Older
-games are using the header entry at `$014B` instead.
+These bytes specify a two-character ASCII licensee code indicating the
+developer/publisher of the game. The New Licensee Code is present in newer
+cartridges only (those released after the SGB was released).
+Older cartridges use the Old Licensee Code at `$014B` instead.
```

Comment:
```suggestion
This area contains a two-character ASCII "licensee code" indicating the game's publisher.
It is only meaningful if the [Old Licensee Code](<#014B – Old Licensee Code>) is exactly $33 (which is essentially all games made after the SGB was released); otherwise, the old code must be considered.
```